doc/api/GVT-g_api.rst

```diff
@@ -0,0 +1,114 @@
+.. _GVT-g_api:
+
+===============
+ACRN GVT-g APIs
+===============
+
+GVT-g is the GPU virtualization solution of Intel, it is open source and
+up-streamed to the Linux kernel. Its implementation over KVM is named KVMGT,
+over Xen is named XenGT and over ACRN is named ACRN-GT. GVT-g could exports
+multiple virtual GPU (alias vGPU) instances for virtual machine system (a.k.a.
+VM). A VM could be assigned one instance of vGPU. The guest OS graphic driver
+need minor modification to drive the vGPU adapter in VM. Every vGPU instance
+will adopts the full HW GPU’s accelerate capability for 3D render and display.
+In the following document, ACRN-GT refers to the glue layer between ACRN
```

Comment:
Add a blank line before this and change to:

    In the following API documentation, ...

``between the ACRN``